client/src/views/login/Login.js

```diff
@@ -20,7 +20,7 @@ function Login() {
 
   return (
     <div>
-      <h1>Customer Login</h1>
+      <h1> Login</h1>
```

Comment:
There's a space in front of the word Login, intended?

Reply:
No, not intended.